source/win-main/DocumentTabs.vue

```diff
@@ -457,7 +477,7 @@ body.darwin {
 
 body.win32 {
   div#tab-container {
-    border-bottom: none;
+    border-top: none;
```

Comment:
Ah alright, here's the reset — but I don't see a corresponding reset on the macOS section, right?